src/Cron_Command.php

```diff
@@ -345,12 +365,13 @@ public function run_now( $args ) {
 
 		$container = $this->site_php_container( $result->site_url );
 		$command   = $result->command;
+		$user      = empty( $result->user ) ? 'root' : $result->user;
```

Comment:
If it is empty then skip adding the user itself, don't make it root, let it be handled by `docker exec` which picks up the default user of the container. For example, in `php` container `www-data` is the default user, so not specifying the user should run the command through `www-data` only and not `root`.

Reply:
ofelia by default executes command as root in all crons. We want to preserve that behaviour here for consistency that's why I've added it. 

While adding cron after site create, there was a weird bug where cron was executing through run-now, but not through ofelia cron. It was because the cron was failing as it was being run as root but was running with run-now as it was being run with `www-data` user.

Hence to prevent such subtle bugs in future, cron job must run as root by default